File: docflow-backend/src/main/java/com/brutecx/docflow_backend/api/dto/audit/SensitiveAccessAuditCursorPageDTO.java
package com.brutecx.docflow_backend.api.dto.audit;

import java.time.Instant;
import java.util.List;
import java.util.UUID;

public record SensitiveAccessAuditCursorPageDTO(
        List<SensitiveAccessAuditDTO> items,
        boolean hasMore,
        Instant nextCursorTimestamp,
        UUID nextCursorId
) {
}

File: docflow-frontend/types/api/SensitiveAccessAuditCursorPageDTO.ts
import { SensitiveAccessAuditRow } from "./SensitiveAccessAuditRow";

export type SensitiveAccessAuditCursorPageDTO = {
  items: SensitiveAccessAuditRow[];
  hasMore: boolean;
  nextCursorTimestamp: string | null;
  nextCursorId: string | null;
};
